src/app/icd/client/DefaultICDClientStorage.h

```diff
@@ -130,6 +130,7 @@ class DefaultICDClientStorage : public ICDClientStorage
         kMonitoredSubject = 5,
         kAesKeyHandle     = 6,
         kHmacKeyHandle    = 7,
+        kClientType       = 8,
```

Comment:
please also update MaxICDClientInfoSize in https://github.com/project-chip/connectedhomeip/blob/master/src/app/icd/client/DefaultICDClientStorage.h#L159

Reply:
done